### ISSUE TITLE
Talkback is not announcing the selected drop down menu item information when aria-label is specified

### ISSUE THREAD

**Issue by HagarSheffer** [microsoft/fluentui#20785]
### Environment Information

- **Package version(s)**: 8.17.2
Device: Samsung S20.
Android version: 11.
Teams version: 1416/1.0.84.2021561002/0909(Canary).
Screen reader: Talkback(9.1).

### Describe the issue:

Talkback is not announcing the selected drop down menu item information when aria-label is specified

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/hagar21/pen/rNzXaBv

#### Actual behavior:

Talkback is not announcing the selected drop down value when focus lands on the drop down. Talkback is only announcing "Collapsed,<Label> Drop down menu"

#### Expected behavior:

Talkback should be announcing the selected drop down value when focus lands on the drop down. Talkback should be announcing "Collapsed,<Label>, <Selected value info>, Drop down menu".

### Documentation describing expected behavior

MAS 1.3.1 – Info and Relationships -
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B97358E88-4367-4121-BF9D-F0C6597D03B7%7D&file=MAS%201.3.1%20%E2%80%93%20Info%20and%20Relationships%20.docx&wdLOR=cC67AE285-FA8E-4A9C-8615-42813289BF4A&action=default&mobileredirect=true&cid=0e16243a-5e31-4456-9293-a60d0a2f5cf8

**Comment by gouttierre** [comment]
@HagarSheffer - Thanks for filing this issue with us. To set expectations, the developers will review the issue once capacity allows.

@spmonahan - Would you be able to confirm if this is a regression, or if this behavior is an issue? 
cc @smhigley
